mindsdb/integrations/handlers/anthropic_handler/README.md

```diff
@@ -0,0 +1,24 @@
+# Anthropic Handler
+Anthropic ML handler integrates Anthropic LLMs with MindsDB. You can train your existing text data with Anthropic LLM
+
+# Anthropic
+Anthropic is a AI assitant framework for your usecases. In this handler, we use the Anthropic package, which is available in Python. More information about this Python client can be found (here)[https://github.com/anthropics/anthropic-sdk-python].
+
+
+# Example Usage
+~~~ sql
+CREATE MODEL mindsdb.anthropic_test
```

Comment:
Please add the prerequisite `CREATE ML_ENGINE` command here.

Reply:
Thanks for the review, I have made the necessary changes